apps/studio/src/features/dashboard/components/CollectionTable/CollectionTable.tsx

```diff
@@ -26,7 +27,11 @@ const getColumns = ({ siteId }: CollectionTableProps) => [
         siteId={siteId}
         id={row.original.id}
         title={row.original.title}
-        permalink={`/${row.original.permalink}`}
+        permalink={
+          row.original.type === ResourceType.CollectionLink
+            ? ""
+            : `/${row.original.permalink}`
+        }
```

Comment:
nit - might wanna pass in `null` and then not rendered as the `VStack` it is in will still render this empty string permalink, causing a "0.25rem margin-bottom" look to the title and it being non-centered with the left icon

https://github.com/opengovsg/isomer/blob/1a66183a6b6cf411f9fc69880619fb56258555df/apps/studio/src/features/dashboard/components/ResourceTable/TitleCell.tsx#L65-L86

Reply:
good catch, lemme update it and try!